src/styles/blocks/card.scss

```diff
@@ -0,0 +1,124 @@
+.card {
+  margin: 0;
+  font-family: Roboto, serif;
+  text-align: left;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  width: 200px;
+
+  border: 1px #f3f3f3 solid;
+
+  transition: transform $transition-time;
```

Comment:
The variable `$transition-time` is used here, but it's not defined within this file. Ensure that `$transition-time` is defined in a global variables file or imported correctly.